packages/components/src/components/CodeEditor.tsx

```diff
@@ -145,18 +151,33 @@ export const CodeEditor: FC<CodeEditorProps> = ({
   }, [onChange]);
 
   useEffect(() => {
+    const cHeight = oneLine
+      ? (() => {
+          if (editorView.current) {
+            const paddings = editorView.current.documentPadding;
+            return (
+              editorView.current.defaultLineHeight +
+              paddings.top +
+              paddings.bottom
```

Comment:
This seems hacky =) But nice that it works.

One minor thing that annoys me in the current behavior (both in this PR and before) is that on load the editor has no height, which leads to some reflow, e.g. on the [playground page](https://squiggle-website-git-cab-autocomplete-quantified-uncertainty.vercel.app/playground) you can initially see the footer and then it gets pushed down by the editor.

Since we often know the height in advance, should we set it as `minHeight` on the div container, maybe?

Reply:
Did that for `height` and `width` constraints, but not for `oneLine` — that one might require a bit of digging into what exactly is the default for line height in codemirror)

But that does fix the problem you've mentioned — now the only blinking component is logo, which is weirdly enough doesn't cache.